docker/docker-compose.suite-desktop-ci.yml

```diff
@@ -14,3 +15,9 @@ services:
     command: bash -c "yarn workspace @trezor/integration-tests test:desktop"
     volumes:
       - ../:/trezor-suite
+
+  electrum-regtest:
+    image: ghcr.io/vdovhanych/electrs:latest
```

Comment:
Use this image instead, please - `ghcr.io/trezor/electrs:latest`

Reply:
[ff2ee66](https://github.com/trezor/trezor-suite/pull/6258/commits/ff2ee66c35ff8a04f1f0477b2b4581aa1347f8f7)